sonar-project.properties

```
@@ -0,0 +1,3 @@
+sonar.projectKey=pagopa_developer-portal
+sonar.organization=pagopa
+sonar.sources=apps/**/src, packages/gitbook-docs/src
```

Why not as following?

```suggestion
sonar.sources=apps/**/src, packages/**/src
```

---

I wasn't sure that running the scan on `eslint-config` package would give useful information.